gtsam/hybrid/GaussianHybridFactorGraph.cpp

```diff
@@ -115,6 +115,19 @@ pair<GaussianMixture::shared_ptr, boost::shared_ptr<Factor>> EliminateHybrid(
     return hasNull?GaussianFactorGraph():gfg;
   };
 
+  // TODO(fan): Now let's assume that all continuous will be eliminated first!
+  // Here sum is null if remaining are all discrete
+  if (sum.empty()) {
+    // Not sure if this is the correct thing, but anyway!
+    DiscreteFactorGraph dfg;
+    dfg.push_back(factors.discreteGraph());
+
+    auto dbn = dfg.eliminatePartialSequential(ordering, EliminateForMPE);
```

Comment:
Actually, *within* eliminateHybrid you should probably just call EliminateFirMPE? Eliminate-sequential is the outer elimination loop, whereas Dliminate does one step (possibly of multiple variables at a a time, but we will only test that when we do multi frontal/Bayes trees)

Reply:
I see, I used that initially but that one has a less easy to use return type, so I switched...